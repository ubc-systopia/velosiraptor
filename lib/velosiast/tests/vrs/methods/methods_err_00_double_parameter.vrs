// method with pre-condition
fn foo(foo: int, bar: int, foo: int) -> bool
{
    true
}

// method with pre-condition
fn foo(foo: int, bar: int, foo: bool) -> bool
{
    true
}
