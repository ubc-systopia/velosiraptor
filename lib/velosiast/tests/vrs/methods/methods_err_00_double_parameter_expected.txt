error: duplicate definitions with name `foo`
     --> tests/vrs/methods/methods_err_00_double_parameter.vrs:2:28
      |
    2 |  fn foo(foo: int, bar: int, foo: int) -> bool
      |         ^^^^^^^^ previous definition of `foo` was here
   ...
      |
    2 |  fn foo(foo: int, bar: int, foo: int) -> bool
      |                             ^^^^^^^^ duplicate definition
error: could not compile due to 1 previous errors; 0 warnings emitted


@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

error: duplicate definitions with name `foo`
     --> tests/vrs/methods/methods_err_00_double_parameter.vrs:8:28
      |
    8 |  fn foo(foo: int, bar: int, foo: bool) -> bool
      |         ^^^^^^^^ previous definition of `foo` was here
   ...
      |
    8 |  fn foo(foo: int, bar: int, foo: bool) -> bool
      |                             ^^^^^^^^^ duplicate definition
error: could not compile due to 1 previous errors; 0 warnings emitted


@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

